test/schema-special-usage.test.js

```diff
@@ -45,6 +45,53 @@ test('Ajv8 usage instead of the bundle one', t => {
       t.match(err.message, 'strictRequired', 'the new ajv8 option trigger a startup error')
     })
   })
+
+  t.test('use new ajv8 option with JTD schema fails (JTD needs explicit dependency)', t => {
```

Comment:
Sorry for the cross comment

https://github.com/fastify/fastify/pull/3096#discussion_r658266290

Reply:
I went ahead and removed that, and added a couple of tests that just show use JTD by explicit inclusion.